README.md

```diff
@@ -428,8 +428,7 @@ a 👍. This helps maintainers prioritize what to work on.
 For questions related to using the library, please visit a support community
 instead of filing an issue on GitHub.
```

Comment:
(Unrelated, but I think we're missing an "l" here)

```suggestion
instead of filling an issue on GitHub.
```

Reply:
I think the usage here is intentionally "filing" as in "to file paperwork". "filling" would technically also work in this context but I feel like it has different semantics, it's more about completing an issue than submitting it.

Reply:
oh, right, that makes sense. thanks!